lib/alchemy/resources_helper.rb

```diff
@@ -91,20 +89,20 @@ def render_attribute(resource, attribute, options={})
     def resource_attribute_field_options(attribute)
       options = {hint: resource_handler.help_text_for(attribute)}
       case attribute[:type].to_s
-      when 'boolean'
-        options
-      when 'date', 'datetime'
-        options.merge as: 'string',
-          input_html: {
-            type: 'date',
-            value: l(resource_instance_variable.send(attribute[:name]) || Time.now, format: :datepicker)
-          }
-      when 'time'
-        options.merge(as: 'time')
-      when 'text'
-        options.merge(as: 'text', input_html: {rows: 4})
+        when 'boolean'
+          options
+        when 'date', 'datetime'
+          options.merge as: 'string',
+            input_html: {
+              type: 'date',
+              value: l(resource_instance_variable.send(attribute[:name]) || Time.now, format: :datepicker)
```

Comment:
Metrics/LineLength: Line is too long. [106/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)